src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -154,6 +158,9 @@ class TestCaseDisplayService(private val project: Project) {
                             HighlighterLayer.FIRST
                         )
                     }
+
+                    // select checkbox
+                    checkbox.isSelected = true
```

Comment:
why do you select the checkbox upon reset?

Reply:
That's not upon reset, that's upon modifying the editor